src/wp-includes/rest-api/endpoints/class-wp-rest-widget-types-controller.php

```diff
@@ -397,6 +420,71 @@ public function get_item_schema() {
 		return $this->add_additional_fields_schema( $this->schema );
 	}
 
+	public function render( $request ) {
+		return array(
+			'preview' => $this->render_legacy_widget_preview_iframe(
+				$request['id_base'],
+				$request['instance']
```

Comment:
```suggestion
				isset($request['instance']) ? $request['instance'] : null
```
As far as I understand `instance` request parameter doesn't have a default value, so I would check if it is present in the `$request` array before using it.
The `instance` array key could be undefined, and this can generate a PHP warning.
Alternatively, we could define a default value for the `instance` parameter here: https://github.com/WordPress/wordpress-develop/blob/b0b6cb921792254fd9ba4d93a2263c17638d0670/src/wp-includes/rest-api/endpoints/class-wp-rest-widget-types-controller.php#L118